deploy.yaml

```diff
@@ -124,4 +113,8 @@
       tags: 100k
     - role: kafka2ftp
     - role: kafka-loopback
-      tags: [ never, 100k ]
+      tags: [never, 100k]
+    - role: kafka-ws
+      tags: [100k]
+    - role: x509-identity-mgmt
+      tags: 100k
```

Comment:
When you don't set a tag, x509-identity-mgmt should be deployed?

Reply:
Yes, it is the substitute for the old EJBCA.